bin/lint-staged.js

```diff
@@ -26,15 +26,16 @@ const version = packageJson.version
 cmdline
   .version(version)
   .option('--allow-empty', 'allow empty commits when tasks revert all staged changes', false)
-  .option('-c, --config [path]', 'path to configuration file, or - to read from stdin')
-  .option('-d, --debug', 'print additional debug information', false)
-  .option('--no-stash', 'disable the backup stash, and do not revert in case of errors', false)
   .option(
     '-p, --concurrent <parallel tasks>',
-    'the number of tasks to run concurrently, or false to run tasks serially',
+    'the number of tasks to run concurrently, or false for serial',
     true
   )
-  .option('-q, --quiet', 'disable lint-staged’s own console output', false)
+  .option('-c, --config [path]', 'path to configuration file, or - to read from stdin')
+  .option('--cwd [path]', 'run all tasks in specific directory, instead of the current')
+  .option('-d, --debug', 'print additional debug information', false)
+  .option('--no-stash', 'disable the backup stash, and do not revert in case of errors', false)
+  .option('-q, --quiet', "disable lint-staged's own console output", false)
```

Comment:
Let's use `’` please and revert other related changes. 

BTW last time I tried using 🚫💩 lint-staged I was wondering what's the difference between `debug` and `quite`. Sounds like mutually exclusive things to me.